docs/tutorial/part-three/index.md

```diff
@@ -232,7 +232,7 @@ const ListLink = props => (
 // highlight-end
 
 export default ({ children }) => (
-  <div style={{ margin: `0 auto`, maxWidth: 650, padding: `1.25rem 1rem` }}>
+  <div style={{ margin: `3rem auto`, maxWidth: 650, padding: `0 1rem` }}>
```

Comment:
Ops, I thought we branched out of master, apparently we did something wrong then, but let's remove this change from this PR

Reply:
I missed to switch the branch. Sorry. I'll fix that.

Reply:
Updated the pull request.